ui/packages/statement/src/components/StatementSummaryTable.tsx

```diff
@@ -1,11 +1,14 @@
 import React from 'react'
 import { Table } from 'antd'
 import moment from 'moment'
-import { StatementDetailInfo } from './statement-types'
 import { useTranslation } from 'react-i18next'
-import SyntaxHighlighter from 'react-syntax-highlighter'
-import { atomOneLight } from 'react-syntax-highlighter/dist/esm/styles/hljs'
 import sqlFormatter from 'sql-formatter-plus'
+import { Light as SyntaxHighlighter } from 'react-syntax-highlighter'
+import sql from 'react-syntax-highlighter/dist/esm/languages/hljs/sql'
+import { atomOneLight } from 'react-syntax-highlighter/dist/esm/styles/hljs'
```

Comment:
maybe `import 'react-syntax-highlighter/dist/esm/styles/hljs/atomOneLight'` is better?

Reply:
`<SyntaxHighlighter language="sql" style={atomOneLight}>` this place need it.

Reply:
but I can try whether we can do that then we don't declare style for SyntaxHighlighter

Reply:
how about `import atomOneLight from 'react-syntax-highlighter/dist/esm/styles/hljs/atomOneLight'`

Reply:
using `import { xxx }` will include all styles I believe.

Reply:
make sense, let me have a try.

Reply:
The 2 ways have the same bundle size after trying.